spec/unit/saver_spec.rb

```diff
@@ -123,14 +123,20 @@
   end
 
   context 'when saving a screenshot fails' do
+    it 'does not raise' do
+      expect(capybara_mock).to receive(:save_page).and_raise
```

Comment:
Why are you raising an error on `save_page` when `current_path` is the method that is raising an exception?